cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.13.1
+
+_Released 06/06/2023 (PENDING)_
+
+**Dependency Updates:**
+
+- Upgraded [`find-process`](https://www.npmjs.com/package/find-process) from `1.4.1` to `1.4.7`. Addressed in [#26631](https://github.com/cypress-io/cypress/pull/26906).
```

Comment:
```suggestion
- Upgraded [`find-process`](https://www.npmjs.com/package/find-process) from `1.4.1` to `1.4.7` to address xxx security vulnerability. Addressed in [#26631](https://github.com/cypress-io/cypress/pull/26906).
```

Reply:
@emilyrohrbough There's not a CVE available. I'm going off of this link - is this an appropriate link for linking to security vulnerabilities? https://security.snyk.io/vuln/SNYK-JS-FINDPROCESS-1090284

Reply:
I think this would work fine!